momento-sdk/src/main/java/momento/sdk/config/Configuration.java

```diff
@@ -11,6 +12,8 @@ public class Configuration {
   private final TransportStrategy transportStrategy;
   private final RetryStrategy retryStrategy;
 
+  private final Integer concurrencyLimit;
```

Comment:
if we decide to push forward with this we should spend a minute thinking about whether this should be a top-level config setting or if it belongs on one of the inner objects. i don't have an opinion yet, just want to be deliberate about it.

Reply:
Agreed.